requires-ci.txt

```diff
@@ -3,24 +3,24 @@ black==22.3.0;python_version>="3.7"
 black==21.6b0;python_version<"3.7"
 dash-flow-example==0.0.5
 dash-dangerously-set-inner-html
-flake8==3.9.2
+flake8==7.0.0
 flaky==3.7.0
 flask-talisman==1.0.0
 isort==4.3.21;python_version<"3.7"
 mimesis<=11.1.0
 mock==4.0.3
-numpy<=1.25.2
+numpy<=1.26.3
 orjson==3.5.4;python_version<"3.7"
-orjson==3.6.7;python_version>="3.7"
+orjson==3.9.12;python_version>="3.7"
```

Comment:
We can get rid of all the `python_version>="3.7"` and `python_version>="3.8"` specifiers, and the entire lines for `<"3.7"` and `<"3.8"`

Reply:
@alexcjohnson ok done!